<ui version="4.0" >
 <class>savedSurfaces</class>
 <widget class="QWidget" name="savedSurfaces" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QListWidget" name="savedSurfacesList" >
     <property name="iconSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="layoutMode" >
      <enum>QListView::Batched</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Double-click on a surface to load it.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QPushButton" name="saveButton" >
     <property name="text" >
      <string>Save current Surface to File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="removeFromListButton" >
     <property name="text" >
      <string>Remove Surface from List</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="addToListButton" >
     <property name="text" >
      <string>Add Files to List</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections/>
</ui>
